src/main/java/uk/gov/justice/hmpps/prison/api/model/OffenderEmploymentResponse.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@ApiModel(description = "Offender Employment")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class OffenderEmploymentResponse {
+
+    @ApiModelProperty(value = "Offender booking id.", example = "14")
+    private Long bookingId;
+
+    @ApiModelProperty(value = "Start date of employment")
+    private LocalDate startDate;
+
+    @ApiModelProperty(value = "End date of employment")
+    private LocalDate endDate;
+
+    @ApiModelProperty(value = "The employment post type condition")
+    private String postType;
+
+    @ApiModelProperty(value = "The obfuscated name of the employer")
+    private String employerName;
+
+    @ApiModelProperty(value = "The obfuscated name of the supervisor in the employment")
+    private String supervisorName;
+
+    @ApiModelProperty(value = "Position held on job")
+    private String position;
+
+    @ApiModelProperty(value = "The reason for leaving job")
+    private String terminationReason;
+
+    @ApiModelProperty(value = "Amount the offender was earning")
+    private Double wage;
+
+    @ApiModelProperty(value = "The frequency of wage payments")
+    private String wagePeriod;
+
+    @ApiModelProperty(value = "The occupation name of the offender")
+    private String occupation;
```

Comment:
examples for these fields (occupation/position/schedule) would be good too?  

Might be worth saying the relationship to reference data - will it be one of a set of values under a specific domain? 

The domains are exposed as part of the API and it can help consumers understand that these values are constrained to a specific set of values rather than user entered